### PR TITLE
reorganized platform section

### PR DESCRIPTION
**Pull request by yannski**

### CODE REVIEW

File: redirections.yml [hunk 1]
```diff
@@ -1,4 +1,46 @@
 "301":
+  -
+    old: "/languages/ruby/private-gem-from-git-on-scalingo/"
+    new: "/languages/ruby/private-gem-from-git-on-scalingo"
+  -
+    old: "/platform/app/transfer-ownership"
+    new: "/platform/app/lifecycle-management#transfer-ownership-of-an-application"
+  -
+    old: "/platform/app/application-lifecycle-management"
```
Comment (Soulou) [inline redirections.yml]: Double `old`

Comment (yannski) [reply redirections.yml]: negative

Comment (yannski) [reply redirections.yml]: whoops, you're right

File: redirections.yml [hunk 2]
```diff
@@ -1,4 +1,46 @@
 "301":
+  -
+    old: "/languages/ruby/private-gem-from-git-on-scalingo/"
+    new: "/languages/ruby/private-gem-from-git-on-scalingo"
+  -
+    old: "/platform/app/transfer-ownership"
+    new: "/platform/app/lifecycle-management#transfer-ownership-of-an-application"
```
Comment (Soulou) [inline redirections.yml]: Invalid destination:

https://documentation-service-pr279.scalingo.io/platform/app/lifecycle-management#transfer-ownership-of-an-application

Comment (Soulou) [reply redirections.yml]: Fixed in previous push

**Comment by Soulou** [comment]
good job LGTM

**Pull request merged**